src/components/Dropdown/Dropdown.jsx

```diff
@@ -96,6 +97,11 @@ const Dropdown = ({
     return {};
   }, [selectedOptions]);
 
+  const ariaLabel = useMemo(() => {
+    return `${readOnly ? "Readonly " : ""} ${tooltipContent} ${
+      selectedOptions.length ? `Selected: ${selectedOptions.map(o => o.label).join(", ")}` : "Select"
+    }`;
+  }, [readOnly, selectedOptions, tooltipContent]);
```

Comment:
Not sure about this one... Better to discuss with Talya? 
Still better then it was before though...